packages/block-library/src/post-comment-content/index.php

```diff
@@ -18,10 +18,14 @@ function render_block_core_post_comment_content( $attributes, $content, $block )
 		return '';
 	}
 	$wrapper_attributes = get_block_wrapper_attributes();
+	$comment_text       = get_comment_text( $block->context['commentId'] );
+	if ( ! $comment_text ) {
+		return;
+	}
 	return sprintf(
 		'<div %1$s>%2$s</div>',
 		$wrapper_attributes,
```

Comment:
Nit: I think we can call `get_block_wrapper_attributes()` directly here. No need to assign it to a variable if we use it only in one place.

Reply:
Done. thanks for the review George!